.gitattributes

```diff
@@ -2,5 +2,7 @@ atrium-api/src/app.rs linguist-generated=true
 atrium-api/src/app/** linguist-generated=true
 atrium-api/src/com.rs linguist-generated=true
 atrium-api/src/com/** linguist-generated=true
+atrium-api/src/tools.rs linguist-generated=true
+atrium-api/src/tools/** linguist-generated=true
```

Comment:
I forgot to add these when I made the changes that created these files earlier. Thank you very much!